config.json

```diff
@@ -3,7 +3,8 @@
     "clusters": {
         "cluster-foo": {
             "region": "eu-west-1",
-            "cluster_id": "72d858c1-e310-42a1-a2a4-8ad7a0230087"
+            "cluster_id": "72d858c1-e310-42a1-a2a4-8ad7a0230087",
+            "macros": "macros/macro_cluster.txt"
```

Comment:
Rename the default extension to be `.macro`
So the file would become something like `macros/defaults.macro`